src/key_value_store.js

```diff
@@ -289,6 +289,12 @@ export class KeyValueStore {
                 if (this.storeName) storesCache.remove(this.storeName);
             });
     }
+
+    getPublicUrl(fileName) {
+        ensureTokenOrLocalStorageEnvExists('key value store');
+        const publicUrl = `https://api.apify.com/v2/key-value-stores/${this.storeId}/records/${fileName}`;
```

Comment:
Let's move the first part that is constant to a `const` at the beginning of the file. We can then replace it with the env var as @mtrunkat  suggested once we have it.